genmodules.py

```diff
@@ -103,6 +104,13 @@ def version(self):
     def is_dkms(self):
         return 'dkms' in self.name
 
+def get_stream_hash(name, stream, version, distro):
+    uniq_str = name + stream + version + distro
+    hash_str = hashlib.md5(uniq_str.encode('utf-8')).hexdigest()[:12]
```

Comment:
@kmittman as @jeremycline mentioned in ,https://github.com/NVIDIA/yum-packaging-precompiled-kmod/issues/16#issuecomment-731413363, 10 to 12 chars is allowed atm, but the v3 looks to limit it to 10 only. perhaps using 10 here will make it more future proof? (but i am no expert at all)

Reply:
Thank you @stdweird I misinterpreted that to indicate 12 characters now, a larger limit later. I'll change it to 10 characters, please let me know if there is anything else you notice.